docs/content.zh/docs/ops/state/task_failure_recovery.md

```diff
@@ -199,6 +199,40 @@ Python API 不支持。
 {{< /tab >}}
 {{< /tabs >}}
 
+#### 示例
```

Comment:
Giving an example in the doc because there is user feedback `Exponential Delay Restart Strategy is complex than Fixed Delay` in the mail list[1]. 

I feel adding this example is useful to let users understand `Exponential Delay Restart Strategy`.

[1] https://lists.apache.org/thread/6glz0d57r8gtpzq4f71vf9066c5x6nyw